qiskit/circuit/random/utils.py

```diff
@@ -47,7 +47,10 @@ def random_circuit(
 
     Args:
         num_qubits (int): number of quantum wires
-        depth (int): layers of operations (i.e. critical path length)
+        depth (int): layers of operations (i.e. critical path length).
+            Note - It is possible that for some of the layers some of the qubits
+            are not a part of any gate, which at the end produces a circuit
+            with lesser critical path.
```

Comment:
I would also give a bit of the why here, and slightly rephrase to: 

" Note that due to the stochastic nature of the method, some of the qubits in some layers may end up not being assigned to any gate, leading to a circuit with a shorter critical path than specified by `depth`."

Reply:
Thanks @eliarbel , please check now.